src/components/Header/Header.styled.ts

```diff
@@ -0,0 +1,39 @@
+import styled from '@emotion/styled';
+import { Flex } from '../common';
+
+export const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  width: 100%;
```

Comment:
```suggestion
  width: 100%;
  max-width: 1440px;
```
p2: width를 100%로 주는 것은 좋지만 max-width 사이즈는 주는 것이 좋겠어요!
다른 대중적인 사이트를 레퍼런스로 보아도 헤더의 max-width를 모두 주고 있습니다.
사용자의 모니터 화면은 모두 다를테니 와이드 모니터를 쓰는 사람에게 현재의 헤더는 조금 불편할 것 같아요!

Reply:
그 부분은 고려하지 못했습니다! 좋은 의견 감사합니다! 바로 반영하도록 하겠습니다!